docs/source/features/parallelisms.rst

```diff
@@ -185,7 +247,8 @@ The NeMo implementation of Sequence Parallelism utilizes functionality from Mega
 Context Parallelism
 ^^^^^^^^^^^^^^^^^^^
 
-Context Parallelism (CP) is a method for parallelizing the processing of neural network activations across multiple GPUs, focusing on the sequence dimension of the input data. Unlike Sequence Parallelism (SP) that only partitions specific types of activations, CP divides all network activations along the sequence dimension.
+Context Parallelism (CP) is a method for parallelizing the processing of neural network activations across multiple GPUs, partitioning the input tensors in the sequence dimension.
+Unlike Sequence Parallelism (SP) that partitions the activations of specific layers, CP divides the activations of all layers.
```

Comment:
Unlike SP, which partitions the activations of specific layers, CP divides the activations of all layers.